config/cert-manager/kustomization.yaml

```diff
@@ -6,19 +6,13 @@ resources:
 helmCharts:
   - name: cert-manager
     repo: https://charts.jetstack.io
-    version: v1.7.1
+    version: v1.12.2
     releaseName: mgc
     namespace: cert-manager
     valuesInline:
       global:
         leaderElection:
           namespace: cert-manager
       installCRDs: true
-      # The Job API is not present
-      startupapicheck:
-        enabled: false
-      # The APIService API is not present
-      cainjector:
```

Comment:
I don't believe we're using the ca injector stuff, or intend to,
but no harm in leaving the installation as the default.